pan_firewall/assets/logs/pan.firewall.yaml

```diff
@@ -230,4 +385,1028 @@ pipeline:
+          name: remap 'pkts' to 'ocsf.traffic.packets'
+          enabled: true
+          sources:
+            - pkts
+          sourceType: attribute
+          target: ocsf.traffic.packets
+          targetType: attribute
+          targetFormat: integer
+          preserveSource: true
+          overrideOnConflict: false
+        - type: attribute-remapper
+          name: remap 'pkts_sent' to 'ocsf.traffic.packets_out'
+          enabled: true
+          sources:
+            - pkts_sent
+          sourceType: attribute
+          target: ocsf.traffic.packets_out
+          targetType: attribute
+          targetFormat: integer
+          preserveSource: true
+          overrideOnConflict: false
+        - type: attribute-remapper
+          name: remap 'pkts_received' to 'ocsf.traffic.packets_in'
+          enabled: true
+          sources:
+            - pkts_received
+          sourceType: attribute
+          target: ocsf.traffic.packets_in
+          targetType: attribute
+          targetFormat: integer
+          preserveSource: true
+          overrideOnConflict: false
+        - type: attribute-remapper
+          name: remap 'chunks' to 'ocsf.traffic.chunks'
+          enabled: true
+          sources:
+            - chunks
+          sourceType: attribute
+          target: ocsf.traffic.chunks
+          targetType: attribute
+          targetFormat: integer
+          preserveSource: true
+          overrideOnConflict: false
+        - type: attribute-remapper
+          name: remap 'chunks_sent' to 'ocsf.traffic.chunks_out'
+          enabled: true
+          sources:
+            - chunks_sent
+          sourceType: attribute
+          target: ocsf.traffic.chunks_out
+          targetType: attribute
+          targetFormat: integer
+          preserveSource: true
+          overrideOnConflict: false
+        - type: attribute-remapper
+          name: remap 'chunks_received' to 'ocsf.traffic.chunks_in'
+          enabled: true
+          sources:
+            - chunks_received
+          sourceType: attribute
+          target: ocsf.traffic.chunks_in
+          targetType: attribute
+          targetFormat: integer
+          preserveSource: true
+          overrideOnConflict: false
+        - type: attribute-remapper
+          name: remap 'network.destination.port' to 'ocsf.dst_endpoint.port'
+          enabled: true
+          sources:
+            - network.destination.port
+          sourceType: attribute
+          target: ocsf.dst_endpoint.port
+          targetType: attribute
+          preserveSource: true
+          overrideOnConflict: false
+        - type: attribute-remapper
+          name: remap 'network.client.port' to 'ocsf.src_endpoint.port'
+          enabled: true
+          sources:
+            - network.client.port
+          sourceType: attribute
+          target: ocsf.src_endpoint.port
+          targetType: attribute
+          preserveSource: true
+          overrideOnConflict: false
```

Comment:
https://github.com/DataDog/integrations-core/pull/21137/files#diff-3301c18057bab7f054ec999849ee3e491c6682432a6b6dea319d238e18e0415bR544

These remappers already exist so you can remove these.